Pop-Grupa-A-Backend/app/main/repositories/task_repository.py

```diff
@@ -1,16 +1,22 @@
 from app.main import db
-from app.main.model import ComputationTask
-from app.main.model.ComputationAccount import ComputationAccount
-
-
-def add_task(task):
-    save_changes(task)
-    return task
+from app.main.model.ComputationTask import ComputationTask
 
 
 def get_tasks_for_user(userId):
-    return ComputationTask.query.filter_by(User_id=userId).all()
+    return ComputationTask.query.filter_by(user_id=userId).all()
+
 
+def update_task(task):
+    try:
+        save_changes(task)
+        return task, 200
```

Comment:
przenieść to do serwisu. To są chyba zmiany dziewczyn, ale jak już wejdą u was to nie będziecie musieli tam sie mergować.

Reply:
ComputationStatus.WORKING.value